src/Integration/Laravel/Cache/src/AsyncAwsDynamoDbStore.php

```diff
@@ -233,13 +234,8 @@ public function add($key, $value, $seconds)
             ]);
 
             return true;
-        } catch (HttpException $e) {
-            $type = $e->getAwsType();
-            if (null !== $type && Str::contains($type, 'ConditionalCheckFailedException')) {
-                return false;
-            }
-
-            throw $e;
+        } catch (ConditionalCheckFailedException $e) {
```

Comment:
You need to bump `"async-aws/dynamo-db": "^1.1"` in composer.json to use this exception
(or let previous code to keep backward compatibility)

Reply:
Done.